doc/api/util.md

```diff
@@ -215,7 +215,7 @@ by `util.inspect`. This is useful for functions like `console.log` or
 set to an object containing one or more valid [`util.inspect()`][] options.
 Setting properties directly is also supported.
 
-Returns the current options as an object.
+Returns the current default options as an object.
```

Comment:
(Whoops, the parenthetical is wrong, ignore it. Also, uh, yeah, looks like I'm commenting on an out-of-date diff. SMH. Sorry.)